counting_prawns.10s.js

```diff
@@ -148,6 +155,8 @@ const bitbarObjectBuilder = (prAssignedToYouDividedByRepos, bitbarConfigVars) =>
 const paintBitBar = () => {
   getPrDividedByReposAndAssignedToYou().then((prAssignedToYou) => {
     bitbar(bitbarObjectBuilder(prAssignedToYou, bitbar));
+  }).catch((errorMessage) => {
```

Comment:
I don't like this solution, you are "unpurify" the function, and as you can see have inputs of different type, force you to implement a cascade of checks for the type. This is just evil, because all the functions here are pure and for the same input return always the same output. I would like to implement a separate scalable method that handle a sort of different errors.